lib/wallaroo/core/autoscale/autoscale_phase.pony

```diff
@@ -534,11 +572,11 @@ class _JoiningWorker is _AutoscalePhase
 
   fun name(): String => "JoiningWorker"
 
-  fun ref worker_join(conn: TCPConnection, worker: WorkerName,
-    worker_count: USize, local_topology: LocalTopology,
-    current_worker_count: USize)
+  fun ref try_join(local_topology: LocalTopologyInitializer,
+    worker_name: WorkerName, worker_count: USize, auth: AmbientAuth,
+    response_fn: TryJoinResponseFn)
   =>
-    None
+    local_topology.worker_join(worker_name, worker_count, response_fn)
```

Comment:
This should be rejected, for the reason that an in-process-of-joining worker isn't ready to receive join requests.

Reply:
It might be worth still implementing `try_join` here and giving a different error message that you've contacted a worker that's not yet part of the cluster.